packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -191,33 +191,6 @@ export class FluidCache implements IPersistedCache {
 				return undefined;
```

Comment:
get() method records telemetry event "FluidCacheAccess".

Can we restructure code such that we record time it takes to open DB & close DB, such that we can see the breakdown of time between opening DB, reading value & closing DB.

Reply:
If we can do that in the same log statement that would be great. Most apps are still running the telemetry pipeline on the main thread

Reply:
Is there any assessment on the cost of telemetry calls? For either type of apps? We might want to revisit guidance in this space depending on the cost, I bet most engineers assume telemetry is cheap.

Reply:
Otel in the default configuration can have up to several ms of overhead per log statement. We're running it in a web worker as an A/B test in Loop app, but that doesn't apply to Outlook, Teams, etc yet.

Reply:
In this PR, not logging any additional telemetry event but just added  some fields in the existng telemetry event.